packages/api/types.ts

```diff
@@ -2061,10 +2061,6 @@ type TSocketEndpoints = {
         request: P2PAdvertiserInformationRequest;
         response: P2PAdvertiserInformationResponse;
     };
-    p2p_advertiser_list: {
-        request: P2PAdvertiserListRequest;
-        response: P2PAdvertiserListResponse;
-    };
```

Comment:
we don't have such endpoint

Reply:
In the old docs they do have it thou 👀 
https://developers.binary.com/api/#p2p_advertiser_list

Reply:
yeahhhh 😓 got confused cause i was looking at the new one 👀